pages/landing/enterprise-eks/index.html

```diff
@@ -219,6 +223,182 @@
           <li>Maintain compliance across CIS benchmark versions</li>
       </ul>
 
+why_gruntwork:
+  - title: Cluster
+    features:
+      - name: "Fully-managed control plane"
+        description: "Provides a Kubernetes control plane that is operated and maintained by a 3rd party as a service (e.g., by AWS)"
+        kops: "no"
+        rancher: "yes"
+        eks: "yes"
+        gruntwork: "yes"
+      - name: "Self-managed control plane"
+        description: "Provides a Kubernetes control plane that your team is responsible for operating and maintaining yourself"
+        kops: "yes"
+        rancher: "yes"
+        eks: "no"
+        gruntwork: "no"
+      - name: "Fully-managed worker nodes"
+        description: "Provides Kubernetes worker nodes that are operated and maintained by a 3rd party as a service (e.g., by AWS)"
+        kops: "no"
+        rancher: "yes"
+        eks: "yes"
+        gruntwork: "yes"
+      - name: "Self-managed worker nodes"
+        description: "Provides Kubernetes worker nodes that your team is responsible for operating and maintaining yourself"
+        kops: "yes"
+        rancher: "yes"
+        eks: "yes"
+        gruntwork: "yes"
+      - name: "Service mesh"
+        description: "Provides a service mesh such as Istio, Consul Connect, or AWS App Mesh out-of-the-box."
+        kops: "no"
+        rancher: "no"
+        eks: "no"
+        gruntwork: "yes"
+  - title: Cluster management
+    features:
+      - name: "Manage everything as code"
+        description: "You can manage your Kubernetes cluster as code using tools such as Terraform or CloudFormation."
+        kops: "yes"
+        rancher: "no"
+        eks: "yes"
+        gruntwork: "yes"
+      - name: "Node auto scaling"
```

Comment:
Isn't there the "HPA" for EKS that allows auto-scaling?

Reply:
Yes, but it's not configured out-of-the-box in many of these solutions; it's extra work you have to do.

In other words, it's _possible_ to get most of these features in most of the solutions, but it requires a ton of extra work. 

I tried to make this a tiny bit clearer: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/600/commits/3cdaf5679c38c795e671f2471c457f464054fcfc